mobile_app_connector/models/request.py

```diff
@@ -65,6 +67,50 @@ def mobile_contact_us(self, parameters, **params):
                            subject=_("Original request from %s %s ") %
                            (firstname, lastname))
 
+        partner = self.env['res.partner'].browse(int(contact_id))
+        if partner:
+            self.create_email_for_interaction_resume(subject, question, partner)
+
         return {
             "FeedbackAndContactusResult": _("Your question was well received")
         }
+
+    def create_email_for_interaction_resume(self, subject, body, partner):
+        """
+        Creates a mail to make the new request appear on the interaction resume
+        :param subject: subject of email
+        :param body: body of email
+        :param partner: partner making the request
+        :return: None
+        """
+        mail = self.env['mail.mail'].create({
+            'state': 'sent',
+            # 'recipient_ids': [(4, contact_id)],
+            'subject': subject,
+            'body_html': body,
+            'author_id': partner.id,
+            'email_from': partner.email,
+            'sent_date': datetime.today(),
+            'mail_message_id': self.env['mail.message'].create({
```

Comment:
I believe that at line 66 of the code, you can catch the message that was created in the claim (support request) when the method `message_post` is called by doing `message = claim.message_post(...)`.
Then here you could use it and link to it instead of creating a new duplicate message. `message_id` would be passed to the function as parameter `message_id = message.id`
```suggestion
            'mail_message_id': message_id
```

Reply:
I tried this and the emails did not appear in the interaction resume. I think it's because the mail.messages are different (different models, etc)